build-system-tests/templates/components/react/vite/vite.config.js

```diff
@@ -0,0 +1,24 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import { NodeGlobalsPolyfillPlugin } from '@esbuild-plugins/node-globals-polyfill';
+import { NodeModulesPolyfillPlugin } from '@esbuild-plugins/node-modules-polyfill';
```

Comment:
Do we still need these? Think they were required fo `aws-amplify@v5`

Reply:
Oh yeah this actually isn't being used anywhere just forgot to take it out, removed